packages/cli/src/commands/migrate.ts

```diff
@@ -456,6 +443,31 @@ async function migrate(
   // Process the tasks either in-process or via a worker pool
   const isSingleThreaded = numThreads === 0;
 
+  if (!nonInteractive) {
+    // See if any of the codemods have choices that need to be made by the user
+    for (let i = 0; i < codemodVersions.length; i++) {
+      const codemod = codemodVersions[i];
+      const choices = codemod.choices;
+      if (!choices) continue;
+
+      log(
+        stderr,
+        ['', `Codemod for version  ${green(codemod.version, env)} requires user input:`, ''].join(
+          '\n',
+        ),
+      );
+
+      for (let key of Object.keys(choices)) {
+        const choice = choices[key];
+        if (choice) {
+          const answer = await choice();
+          const setAnswers = codemod.setAnswers?.[key] ?? (() => {});
+          setAnswers(answer);
```

Comment:
Feels like we could use some validation for nonsensical input/validation against the choices options or am I missing that?